src/prototypo-canvas.js

```diff
@@ -502,18 +528,151 @@ PrototypoCanvas.prototype.setupEvents = function( pCanvasInstance ) {
 					[`contours.${contourIdx}.nodes.${nodeIdx}.expand`]: { width: deltaWidth },
 				};
 				this.changesToConfirm = cursors;
-				return emitEvent('manualchange', [ cursors ]);
+				sendManualChanges(cursors);
+				break;
+			case 'skeleton':
```

Comment:
I think it would be clearer to have only one level of switch and different changes type for skeleton (skeleton, skeletonGotSnapped...)